setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name = "python-dispatch",
-    version = "v0.1.2",
+    version = "0.2.0",
```

Comment:
I will happily consider your suggestion after you've answered my question.

Reply:
I already have:

> I realize this causes a normalization warning and it would be better to drop the "v". This change is outside of the scope of this PR though and I would like to keep it as it was

> I prefer to keep version and project metadata changes outside of pull requests.

There is now a [contributing info](https://github.com/nocarryr/python-dispatch/blob/master/CONTRIBUTING.md) document that may be used for reference

Reply:
> I would like to keep it as it was

I will not be giving up my time if that's your "reason" for keeping the v-prefix.

Reply:
Very well.  Best of luck to you.